DependencyInjection/CmfSeoExtension.php

```diff
@@ -197,21 +200,27 @@ private function loadAlternateLocaleProvider($config, ContainerBuilder $containe
             ? $this->defaultAlternateLocaleProviderId
             : $config['provider_id'];
 
+        if (!$alternateLocaleProvider) {
+            return;
+        }
 
-        if ($alternateLocaleProvider) {
-            $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
+        $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
+        if ($container->hasDefinition('cmf_seo.event_listener.seo_content')) {
```

Comment:
i do that below. if i don't check first, i get an error. i could check the value on $config but that seems more complicated to me and has no gain as i still use the service name below.